### ISSUE TITLE
Some basic questions

### ISSUE THREAD

**Issue by pgregg88** [otem/Raspberry-Pi-Looper-synth-drum-thing#4]
Hello Otem. Great work.

I'm attempting to recreate and I'm documenting my steps. I have a few clarifications and a question. Would you like me to publish them here or somewhere else?

1. Are you running Pd 0.48-0 "vanilla" with only the comport and ggee extensions? Or a different version of Pd?

2. On the PRI are you running Raspbian Jessie? (Stretch is now the default for new Raspbian installs).

3. On the Sparkfun push button boards... for the loop control buttons (8 buttons)... Did you cut down the larger 4x4 board or did you use (2) of the 2 x 2s?

4. Currently, I'm stuck getting serial communication working between the Teensy 3.6 and the Rpi. -
- I've identified that my serial port on the Pi os "0" when yours was "1".
- I changed the object in serialControls to read comport 0 9600, but I still get the following error scrolling rapidly in the Pd Console"
[comport]: "Write failed for 0 bytes, error is 11" (once it was error is o).
- I've confirmed the Pi is part of the DIALOUT group that has write privledges to /dev/ttyAMA0

Did you have to stop getty on the serial port or anything like that?

I'll continue to work through this. If you have any tips, I'd appreciate them. Thx.

**Comment by DubbyLabby** [comment]
https://youtu.be/mCOHFyI3Eoo

You can find some info about early projects here
http://www.otemrellik.com/tag/teensy/

Also reach him by twiteer or instagram.

**Comment by castless** [comment]
Hi hows it going? I am also trying to reproduce this project and I need some more info because i am a total novice at doing this type of work, any help would be really appreciated thanks.

**Comment by therightprofile** [comment]
@pgregg88 did you manage to recreate the project by any chance? I am trying to get my head around this as well but I am having a hard time.

**Comment by pgregg88** [comment]
No. I put the project on pause. I wasn’t able to get the pure data parts working cleanly.
